common/views/components/ViewerTopBar/ViewerTopBar.js

```diff
@@ -167,6 +169,27 @@ const ViewerTopBar = ({
         )}
         {enhanced && (
           <div className="flex flex--v-center">
+            {document && document.fullscreenEnabled && (
+              <Space h={{ size: 'm', properties: ['margin-right'] }}>
+                <Button
+                  extraClasses="btn--primary-black"
+                  icon="expand"
+                  text="Full screen"
+                  fontFamily="hnl"
+                  clickHandler={() => {
+                    if (viewerRef && viewerRef.current) {
+                      if (!document.fullscreenElement) {
+                        viewerRef.current.requestFullscreen();
```

Comment:
How cross-browser is this?

> For the moment not all browsers are implementing the unprefixed version of the API (for vendor agnostic access to the Fullscreen API you can use [Fscreen](https://github.com/rafrex/fscreen)).

(From [MDN guide to using fullscreen API](https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API/Guide)).